codility_training/lessons.lesson02.Arrays.OddOccurrencesInArray/sangmandu.py

```diff
@@ -0,0 +1,9 @@
+
+def solution(A):
+	sA = sorted(A)
+	if len(sA) == 0:
+		return sA[2*i]
+	for i in range(len(sA)//2):
+		if sA[2*i] != sA[2*i+1]:
+			return sA[2*i]

```

Comment:
```suggestion
        size = len(sA)
	if size == 0:
		return sA[2*i]
	for i in range(size // 2):
		if sA[2 * i] != sA[2 * i + 1]:
			return sA[2 * i]

```
배열 길이를 계산하는 변수는 미리 저장해놓기!